initial commit

abbyfleming opened the pull request:
initial file structure (html, css). still working on the spacing/logic.

abbyfleming committed:
initial commit

JoeShep commented on quiz.js:
```diff
@@ -0,0 +1,72 @@
+	 if (heightInput && charInput){
+
+		beautifulTree(heightInput, charInput);
+	
```
Note that the requirements state that you need to pass an object into your tree-growing function. So take these two variables and make them properties in an object instead. Then pass that object into your function. Come see me if you need help figuring that out

JoeShep commented on quiz.js:
```diff
@@ -0,0 +1,72 @@
+	// prettySpaces = prettySpaces.repeat(heightInput - i);
+
+	console.log(prettyTree);
+
```
This logic is working great. You just need to add in a way to put a certain number of spaces in front of the characters each time through the loop. You're on the right track with calculating spaces as height minus one for the first line. Then you just keep subtracting one from the previous number of spaces each time through the loop

JoeShep commented on quiz.js:
```diff
@@ -0,0 +1,72 @@
+*/
+
+
+//console.log(theloop());
```
Make sure when you're done you remove all commented out code.

JoeShep reviewed (commented):
Abby, you are right on track so far. Your code shows that you're very close to getting a nice, straight tree printing out. 

You will need to add logic for submitting the user input on press of the enter key.

See my comments below for some additional feedback

JoeShep commented:
@abbyfleming I just want to make sure you're still working on getting this to 100%

abbyfleming committed:
spaces added

abbyfleming commented:
Added spaces, created new object, added event listeners for enter key. (Still slightly confused about when I should be using the ";" but that will come with time and using the linter.)
